keyboards/clickety_split/leeloo/rev1/config.h

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2022 Clickety Split Ltd.
+ *                https://clicketysplit.ca
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1004
```

Comment:
This vendor ID is already in use, and needs to be changed. 

Anything about 0x4000 should be okay, and the higher, the better.

Reply:
Hi @drashna!

I was able to update the VENDOR_ID to 0x4013.  I hope this ID will be okay. :)